### PR TITLE
feat(admin-ui): Add Russian translation

### PR DESCRIPTION
**Pull request by yersultanur**

### CODE REVIEW

**Comment by bytadaniel** [comment]
Hello! I really need this localization for my production project. @olivermrbl Could you, please, tell me, in what time frame do you plan to put together a release with Russian localization?

**Comment by yersultanur** [comment]
@bytadaniel please, if you have time check translation, are you satisfied with it?

**Comment by bytadaniel** [comment]
@yersultanur Hello! Could you please contact me and help me to startup medusa with your russian i18n. I will be glad to check out the translation. I am a russian native speaker

**Comment by yersultanur** [comment]
@bytadaniel [t.me/mirvoinaput]( t.me/mirvoinaput) you contact me through telegram, and I will send you json, so you can take a look on translation

File: packages/admin-ui/ui/src/i18n/index.ts [hunk 1]
```diff
@@ -16,10 +16,14 @@ export const supportedLanguages = [
...
   },
+  {
+    locale: "ru",
+    name: "Русский",
+  },
 ]
 
 const backendUrl = window.location.origin
-const adminPath = process.env.ADMIN_PATH ? `${process.env.ADMIN_PATH}/` : "/"
+const adminPath = process.env.ADMIN_PATH ? `${process.env.ADMIN_PATH}` : "/"
```
Comment (VariableVic) [inline packages/admin-ui/ui/src/i18n/index.ts]: This part is outdated, could you resolve the conflicts please? Then we can merge the translation.

**Review by VariableVic — CHANGES_REQUESTED** [review]

**Comment by yersultanur** [comment]
@VariableVic Hi ,is it good now?

**Review by VariableVic — APPROVED** [review]
Looks good now, thanks! Nice work

**Pull request merged**